examples/aggregator/module/rust/src/lib.rs

```diff
@@ -151,11 +151,10 @@ impl Aggregator for AggregatorNode {
     }
 }
 
-oak::entrypoint!(aggregator => |in_channel| {
+oak::entrypoint!(aggregator<AggregatorInit> => |init_receiver: Receiver<AggregatorInit>| {
```

Comment:
I find it surprising that type inference doesn't work here, seems like a fairly trivial case.